src/components/select-button-group/index.js

```diff
@@ -16,6 +17,7 @@ const SelectButtonGroup = ({
 }) => {
     return values.map(value => (
         <SelectButton
+            key={uuidv4()}
```

Comment:
The problem here would be that it will call uuid everytime this is redrawn, which would cause the key to change (which you don't want either). We need the key's to be static, and locally unique (they don't have to be unique across family's, but do need to be unique among siblings).

One possibility would be to use the value as the key, which hopefully should be unique among siblings (I'm only about 95% confident in that here), and should remain static (I'm also not 100% certain on this).